flambe/runner/utils.py

```diff
@@ -50,6 +53,6 @@ def check_system_reqs() -> None:
 
     # Check if extensions folder is getting big
     extensions_folder = os.path.join(FLAMBE_GLOBAL_FOLDER, "extensions")
-    if get_folder_size_MB(extensions_folder) > WARN_LIMIT_MB:
+    if os.path.exists(extensions_folder) and get_size_MB(extensions_folder) > WARN_LIMIT_MB:
```

Comment:
I don't follow why we are logging this in two different places.

Reply:
Good point.